zipfs/file.go

```
@@ -0,0 +1,164 @@
+package zipfs
+
+import (
+	"archive/zip"
+	"io"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/spf13/afero"
+)
+
+type File struct {
+	fs            *Fs
+	zipfile       *zip.File
+	reader        io.ReadCloser
+	offset        int64
+	isdir, closed bool
+	buf           []byte
+}
+
+func (f *File) fillBuffer(offset int64) (err error) {
+	if f.reader == nil {
+		if f.reader, err = f.zipfile.Open(); err != nil {
+			return
+		}
+	}
+	if offset > int64(f.zipfile.UncompressedSize64) {
+		offset = int64(f.zipfile.UncompressedSize64)
+		err = io.EOF
+	}
+	if len(f.buf) >= int(offset) {
+		return
+	}
+	buf := make([]byte, int(offset)-len(f.buf))
+	n, _ := io.ReadFull(f.reader, buf)
```

Do not ignore the error

---

Fixed.